packages/utils/src/useTableTools/useFilterConfig.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
```

Comment:
This hook is from compliance-frontend: https://github.com/RedHatInsights/compliance-frontend/blob/master/src/Utilities/hooks/useFilterConfig.js